android/app/src/main/java/org/missingmaps/mapswipe/MainApplication.java

```diff
@@ -6,20 +6,10 @@
 import com.facebook.react.PackageList;
 
 import com.facebook.react.ReactApplication;
-import io.invertase.firebase.RNFirebasePackage;
+import com.facebook.react.ReactInstanceManager;
 import com.BV.LinearGradient.LinearGradientPackage;
 import com.reactnativecommunity.asyncstorage.AsyncStoragePackage;
-import io.sentry.RNSentryPackage;
```

Comment:
did you remove sentry as well? or do we still get the errors reports in sentry?

Reply:
See point above: the import system is slightly different in this version of RN, we still very much want sentry to report errors, as we'd be virtually blind without it.